### PR TITLE
Fix Dell U3415W Uniformity Compensation control

### PR DESCRIPTION
**Pull request by SimonAlling**
* Uniformity Compensation can now be disabled.
* The label is now Uniformity Compensation instead of Color Uniformity,
matching the OSD (at least on my U3415W from week 15, 2015).

### CODE REVIEW

**Comment by SimonAlling** [comment]
Thoughts, @jvstein? :slightly_smiling_face:

**Review by larstobi — APPROVED** [review]
LGTM. Should we wait for reply from @jvstein before merging?

**Comment by jvstein** [comment]
@SimonAlling :+1: I'm still on an OS-installed `ddccontrol version 0.4.4` version with my manually added profiles, so I couldn't verify the `coloruniformity` vs `uniformitycompensation` change, but the value matches my monitor's settings.

**Comment by SimonAlling** [comment]
> @SimonAlling +1 I'm still on an OS-installed `ddccontrol version 0.4.4` version with my manually added profiles, so I couldn't verify the `coloruniformity` vs `uniformitycompensation` change, but the value matches my monitor's settings.

Great! I have tested the changes, so I think this PR is ready for merging!

**Pull request merged**
